src/containers/org/MemberRoleCard.js

```diff
@@ -66,21 +30,12 @@ const MemberRoleCard = ({
   };
 
   return (
-    <RoleCard>
-      <Flex>
-        <StyledRoleIcon />
-        <StyledLink to={rolePath}>
-          {role.title}
-        </StyledLink>
-      </Flex>
-      {
-        isUnassignable && (
-          <IconButton aria-label="unassign-role" onClick={handleClick}>
-            <FontAwesomeIcon fixedWidth icon={faTimes} />
-          </IconButton>
-        )
-      }
-    </RoleCard>
+    <Chip
+        clickable
+        component="a"
+        href={`#${rolePath}`}
+        label={role.title}
+        onDelete={isUnassignable ? handleClick : undefined} />
```

Comment:
No change required, just a thought. The logic in this component now looks very similar to the logic in `RoleChipsList` and `RoleOverflowPopover`. This seems like a good candidate for a reusable component, something like `RoleChip`, that can be used in `MemberRolesContainer` as well as in `PeopleTable`.